desktop/mac/release-notes/index.md

```diff
@@ -33,6 +33,38 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 > chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){:
 > .button .primary-btn }
 
+### New
+
+- Virtiofs released as an experimental feature
+- Added volumes UI bulk removal
+
+### Upgrades
+
+- [Compose v2.3.2](https://github.com/docker/compose/releases/tag/v2.3.2)
+- [containerd v1.4.13](https://github.com/containerd/containerd/releases/tag/v1.4.13)
+- [runc v1.0.3](https://github.com/opencontainers/runc/releases/tag/v1.0.3)
+- [Go 1.17.8](https://golang.org/doc/go1.17)
+- [Linux kernel 5.10.47](https://hub.docker.com/layers/docker/for-desktop-kernel/5.10.92-850089fd41a54a4b84338aa3a118f2197a10ffd9/images/sha256-032232c6e91271035422dc89834fc01339597b0568b4868b1cfeef411a63d17d?context=explore)
+
+### Bug fixes and minor changes
+
+- Improved the error message from `docker build` when using Registry Access Management.
```

Comment:
@djs55 any more details about this? What difference should it make for users?

Reply:
Before it would say something like "I/O timeout". After it says: "Access to ... is restricted by your Docker organization. To access this registry, ask your Docker administrator to add it to your organization's allowed registries."

Perhaps we could say something like
- When Registry Access Management is enabled and a Dockerfile references a blocked registry, the error from `docker build` explains the problem and tells the developer how to resolve it